openapi.yaml

```diff
@@ -0,0 +1,492 @@
+
+tags:
+  - name: "Person"
+    description: "Everything about Persons"
+  - name: "PassportStore"
+    description: "Everything about Passport"
+  - name: "Search"
+    description: "Will be used to find person by passport number"
+  - name: "LostPassport"
+    description: "Will be used to deactivate passport when it was lost"
+
+paths:
+  /person:
+    post:
+      tags:
+        - "Person"
+      summary: "Add a new person to the store"
+      description: "Add new person"
+      operationId: "addPerson"
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/PersonRequest'
+      responses:
+        "200":
+          description: "Successful operation"
+          content:
+            '*/*':
+              schema:
+                $ref: "#/components/schemas/PersonResponse"
+        "400":
+          description: "Invalid data"
+
+  /person/{id}:
+    get:
+      tags:
+        - "Person"
+      summary: "Find person by ID"
+      description: "Returns a single person with current ID"
+      operationId: "getPersonById"
+      parameters:
+        - name: "id"
+          in: "path"
+          description: "Person ID"
+          required: true
+          schema:
+            type: "string"
+      responses:
+        "200":
+          description: "Successful operation"
+          content:
+            '*/*':
+              schema:
+                $ref: "#/components/schemas/PersonResponse"
+        "400":
+          description: "Invalid data"
+        "404":
+          description: "Person not found"
+    put:
+      tags:
+        - "Person"
+      summary: "Updates a person in the store with form data"
+      description: "Updates person details with current ID"
+      operationId: "updatePerson"
+      parameters:
+        - name: "id"
+          in: path
+          description: "Person ID to update"
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/PersonRequest'
+      responses:
+        "200":
+          description: "Successful operation"
+          content:
+            '*/*':
+              schema:
+                $ref: "#/components/schemas/PersonResponse"
+        "400":
+          description: "Invalid data"
+    delete:
+      tags:
+        - "Person"
+      summary: "Delete a person"
+      description: "Delete a person with current ID"
+      operationId: "deletePerson"
+      parameters:
+        - name: "id"
+          in: "path"
+          description: "Person`s ID to delete"
+          required: true
+          schema:
+            type: "string"
+      responses:
+        "200":
+          description: "Successful operation"
+          content:
+            '*/*':
+              schema:
+                $ref: "#/components/schemas/PersonResponse"
```

Comment:
If a person is deleted, it's worth returning 204 without a payload

Reply:
fixed it changed to 204